src/NSwag.CodeGeneration/Models/IOperationModel.cs

```diff
@@ -15,5 +15,8 @@ public interface IOperationModel
     {
         /// <summary>Gets the responses.</summary>
         IEnumerable<ResponseModelBase> Responses { get; }
+
+        /// <summary>Gets Swagger operation's mime type.</summary>
+        string Produces { get; }
```

Comment:
Where is this implemented?

Reply:
It's already implemented by every class that implements this interface.